queries.md

```diff
@@ -12,21 +12,65 @@
  - **`projection`**: {name: 1, _id: 0}
 
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
+{name: 'Twitter'}
+{category_code: 1, name:1, founded_year: 1}
```

Comment:
Here, don't forget to specify "_id: 0", or it's going to be retrieved by default!